app/frontend/src/components/Datepicker.tsx

```diff
@@ -52,12 +51,7 @@ export default function Datepicker({
             }}
             label={label}
             minDate={minDate}
-            onChange={(date) => {
-              //user might be typing, so check the date is valid before doing conversions
-              date?.isValid()
-                ? onChange(date?.format().split("T")[0])
-                : onChange(date);
-            }}
+            onChange={(date) => onChange(date)}
```

Comment:
I wouldn't have commented if there were no other changes, but since there are this can just `onChange={onChange}`

Reply:
You should comment on this even if there's no other changes :P

Thanks for your thorough review!!